senz-web/frontend/src/_actions/auth.js

```diff
@@ -54,6 +54,7 @@ export const LoginAction = ({ email, password }, history) => {
         type: AUTHENTICATION_ERROR,
         payload: "Invalid"
       });
+      alert("Email or password entered is incorrect");
```

Comment:
Can we have a component giving this message instead of an alert ?

Reply:
@YashMeh on it. Thanks for pointing out.